polymer.json

```diff
@@ -1,5 +1,5 @@
 {
   "lint": {
-    "rules": ["polymer-2-hybrid"]
+    "rules": ["polymer-2"]
```

Comment:
Is there no `polymer-3` ruleset yet?

Reply:
There is. I didn't know when I did the d2l-image conversion